policyengine_us/parameters/gov/states/dc/dhcf/medicaid/eligibility/adult_income_limit.yaml

```diff
@@ -0,0 +1,16 @@
+description: The District of Columbia limits Medicaid/Alliance for adults (21+) to this percentage of the Federal Poverty Level.
+metadata:
+  unit: /1
+  period: year
+  label: DC Medicaid/Alliance income limit for adults
+  reference:
+    - title: DC Health Care Alliance - Program Changes
+      href: https://dhcf.dc.gov/node/1796971
+    - title: KFF Medicaid Income Eligibility Limits - DC at 215% FPL for adults
+      href: https://www.kff.org/health-reform/state-indicator/medicaid-income-eligibility-limits-for-adults-as-a-percent-of-the-federal-poverty-level/
+    - title: Fiscal Year 2026 Budget Support Act of 2025 (Bill 26-260)
+      href: https://lims.dccouncil.gov/Legislation/B26-0260
```

Comment:
Is there a primary source which is affected by this bill? Same question for other parameter values